### PR TITLE
[doc] PR for Documentation for Azure KMS

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-4162

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/security/create-kms-config/aws-kms.md [hunk 1]
```diff
@@ -14,14 +14,20 @@ type: docs
...
-      <i class="icon-postgres" aria-hidden="true"></i>
+      <i class="fab fa-aws" aria-hidden="true"></i>
       AWS KMS
     </a>
   </li>
 
+  <li >
+    <a href="{{< relref "./azure-kms.md" >}}" class="nav-link">
+      <i class="icon-azure" aria-hidden="true"></i>&nbsp;
+      &nbsp;&nbsp;Azure KMS
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/security/create-kms-config/aws-kms.md:27]: Do we need this extra spacing?

File: docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+3. Click **Save**.<br>
+
+    Your new configuration should appear in the list of configurations. A saved KMS configuration can only be deleted if it is not in use by any existing universes.
+
+4. Optionally, to confirm that the information is correct, click **Show details**. Note that sensitive configuration values are displayed partially masked.
+
+
+
+Note that YugabyteDB Anywhere does not manage the key vault.
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md:86]: Can we also add a note that says something like "Note: Deleting the KMS config on YBA doesn't delete the actual master key on the KMS provider."

File: docs/content/preview/yugabyte-platform/security/create-kms-config/aws-kms.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/create-kms-config/aws-kms.md:27]: yes, to accommodate an image

**Review by Sahith02 — APPROVED** [review]
Thanks Liza! Let's wait until 2.16 to land this!

File: docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+</ul>
+
+Encryption at rest uses universe keys to encrypt and decrypt universe data keys. You can use the YugabyteDB Anywhere UI to create key management service (KMS) configurations for generating the required universe keys for one or more YugabyteDB universes. Encryption at rest in YugabyteDB Anywhere supports the use of Microsoft Azure KMS. 
+
+Conceptually, Azure KMS consists of a key vault containing one or more keys, with each key capable of having multiple versions.
+
+Before defining a KMS configuration with YugabyteDB Anywhere, you need to create a key vault through the [Azure portal](https://docs.microsoft.com/en-us/azure/key-vault/general/quick-create-portal). The following settings are required:
+
+- Set the vault permission model as Vault access policy.
+- Add the application to the key vault access policies with the minimum key management operations permissions of Get, List, Create, and cryptographic operations permissions of Unwrap Key, Wrap Key.
```
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md:47]: Hey Liza, here are the updated required permissions:
- Key Management Operations - Get
- Key Management Operations - Create (Not required if user is pre-creating the key)
- Cryptographic Operations - Unwrap Key
- Cryptographic Operations - Wrap Key

Can you please update the doc? Thanks

**Review by Sahith02 — CHANGES_REQUESTED** [review]

File: docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/security/create-kms-config/azure-kms.md]: Removed the List permission

**Review by Sahith02 — APPROVED** [review]

**Pull request merged**